pkg/pb/gen.go

```diff
@@ -3,4 +3,4 @@
 
 package pb
 
-//go:generate go run ../../scripts/protobuf.go generate
+//go:generate sh -c "cd ../../scripts; go run ./protobuf generate"
```

Comment:
This will fail on windows :(

Reply:
Dang, because no shell. That's a problem.

Reply:
We can go with this for now.

Reply:
The best I can think of is having a go script that doesn't have any external dependencies and handles `go.mod` related compilation and things. e.g.

```
//go:generate modrun ../../scripts/protobuf generate
```